scripts/install.js

```diff
@@ -0,0 +1,13 @@
+const cp = require('child_process');
+const os = require('os');
+
+// Skip the node-gyp rebuild step when building NPM tfjs-node-gpu package.
+if(!(process.argv[2] === 'compile-npm' && os.platform() === 'linux')) {
```

Comment:
As discussed offline - let's move most of our install logic - see https://github.com/tensorflow/tfjs/issues/601

Reply:
PR description updated with changes.